third_party/3/geoip2/errors.pyi

```diff
@@ -0,0 +1,20 @@
+# Stubs for geoip2.errors (Python 3)
+
+from typing import Optional
+
+class GeoIP2Error(RuntimeError): ...
+
+class AddressNotFoundError(GeoIP2Error): ...
+
+class AuthenticationError(GeoIP2Error): ...
+
+class HTTPError(GeoIP2Error):
+    http_status: int = ...
```

Comment:
Looks like these attributes should be Optional. Also, you can remove the `= ...`.

Reply:
Yep, you're right.